core/src/impl/Kokkos_Combined_Reducer.hpp

```diff
@@ -210,6 +210,8 @@ struct CombinedReducerImpl<std::integer_sequence<size_t, Idxs...>, Space,
      ...);
   }
 
+  KOKKOS_FUNCTION auto& reference() const { return *m_value_view.data(); }
```

Comment:
Where is this used?

Reply:
It's used in where each reducer's value is initialized and passed into provided functor.
For example, for `Cuda`:
https://github.com/kokkos/kokkos/blob/15989dd3a1edb72bbc178fc74d081502e32dd3d6/core/src/Cuda/Kokkos_Cuda_Team.hpp#L630-L649